### ISSUE TITLE
out_sacn: Streaming ACN output

### ISSUE THREAD

**Issue by vifino** [vifino/sled#19]
Outputting sACN frames would be great, as there are many matrix setups in the lighting industry that accept sACN or ArtNet.

The MateLight at GPN 2018 had sACN support, hopefully they'll add this output module. :)

**Comment by vifino** [comment]
@LeoDJ, you're the one who worked on the MateLight, right?

Any chance you're still interested in implementing this? :)

**Comment by LeoDJ** [comment]
Yes, we use ArtNet for our MateLight. And I still want to implement it, but like I said, I sadly don't have that much time.
It is definitely on the top of my to-do list.

**Comment by vifino** [comment]
Ah! Very nice.
I thought it was sACN, my bad.
Don't worry about it, just wanted to make sure I don't forget it or the person who'd use this. :)
